specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-02-01-preview/mfe.json

```diff
@@ -9384,6 +11796,41 @@
       "x-ms-client-name": "BatchDeploymentProperties",
       "additionalProperties": false
     },
+    "BatchDeploymentProperties": {
```

Comment:
Same as `ComputeResource`, other swagger file already contains the model with the same name but with completely different properties.
PLease change a name for this model.

Reply:
I only see this x-ms-client name using the "BatchDeploymentProperties" name for the previous version: https://github.com/Azure/azure-rest-api-specs/blob/653f6cdde3728e5021229d1d78d8699e41d70600/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/mfe.json#L9384